package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-document-generator",
-  "version": "0.10.1",
+  "version": "0.11.2",
```

Comment:
Both minor and patch version have jumped up here, should either be 0.10.2 or 0.11.0

Reply:
done, bumped to 0.11.0 instead